docs/build-notifications.rst

```diff
@@ -20,31 +20,246 @@ Take these steps to enable build notifications using email:
 
 You should now get notified by email when your builds fail!
 
-Using webhook
--------------
+Webhooks
+--------
 
 Read the Docs can also send webhooks when builds are triggered, successful or failed.
 
 Take these steps to enable build notifications using a webhook:
 
-* Go to :guilabel:`Admin` > :guilabel:`Notifications` in your project.
-* Fill in the **URL** field under the **New Webhook Notifications** heading
-* Submit the form
+* Go to :guilabel:`Admin` > :guilabel:`Webhooks` in your project.
+* Fill in the **URL** field and select what events will trigger the webhook
+* Modify the payload or leave the default (see below)
+* Click on :guilabel:`Save`
+
+.. figure:: /_static/images/webhooks-events.png
+   :align: center
+   :alt: URL and events for a webhook
+
+   URL and events for a webhook
+
+Every time one of the checked events triggers,
+Read the Docs will send a POST request to your webhook URL.
+The default payload will look like this:
+
+.. code-block:: json
+
+   {
+       "event": "build:triggered",
+       "name": "Read the Docs",
+       "slug": "rtd",
+       "version": "latest",
+       "commit": "e8dd17a3f1627dd206d721e4be08ae6766fda40",
+       "build": 6321373
+   }
+
+When a webhook is sent, a new entry will be added to the
+"Recent Activity" table. By clicking on each individual entry,
+you will see the server response, the webhook request, and the payload.
+
+.. figure:: /_static/images/webhooks-activity.png
+   :align: center
+   :alt: Activity of a webhook
+
+   Activity of a webhook
+
+Validating the payload
```

Comment:
Yep, although I was hesitant because it's kind of important to avoid attacks I suppose. In any case, I agree that not everybody needs to rush this, so I'll move it down.